Core/navigatorsharepatch.js

```diff
@@ -0,0 +1,39 @@
+ //
+ //  navigatorsharepatch.js
+ //  DuckDuckGo
+ //
+ //  Copyright © 2020 DuckDuckGo. All rights reserved.
+ //
+ //  Licensed under the Apache License, Version 2.0 (the "License");
+ //  you may not use this file except in compliance with the License.
+ //  You may obtain a copy of the License at
+ //
+ //  http://www.apache.org/licenses/LICENSE-2.0
+ //
+ //  Unless required by applicable law or agreed to in writing, software
+ //  distributed under the License is distributed on an "AS IS" BASIS,
+ //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ //  See the License for the specific language governing permissions and
+ //  limitations under the License.
+ //
+
+function isSensitiveFile(filename) {
+    return /^file:\/\/\/(private|var|etc|usr)/i.test(filename);
```

Comment:
I think we should exclude all local files - WebShare should share URIs not whole files AFAIK.
I'd also use an URL object instead of regex to make sure that we are parsing things correctly.

```suggestion
    let uriObj = null;
    try {
       uriObj = new URL(filename);
    } catch (e) {
       return true;
    }
    
    return uriObj.protocol === 'file:';
```